files/zh-cn/learn_web_development/core/structuring_content/emphasis_and_importance/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: 强调与重要性
+slug: Learn_web_development/Core/Structuring_content/Emphasis_and_importance
+l10n:
+  sourceCommit: 5b20f5f4265f988f80f513db0e4b35c7e0cd70dc
+---
+
+{{LearnSidebar}}
+
+{{PreviousMenuNext("Learn_web_development/Core/Structuring_content/Headings_and_paragraphs", "Learn_web_development/Core/Structuring_content/Lists", "Learn_web_development/Core/Structuring_content")}}
+
+上一篇文章探讨了语义在 HTML 中的重要性，重点是标题和段落。本文将继续以语义为主题，探讨对文本进行强调和重要性处理的 HTML 元素（与印刷媒体中的斜体和粗体文字类似）。
+
+<table>
+  <tbody>
+    <tr>
+      <th scope="row">前提：</th>
+      <td>
+        基本熟悉了<a href="/zh-CN/docs/Learn_web_development/Core/Structuring_content/Basic_HTML_syntax"
+          >基本 HTML 语法</a
+        >中所涵盖的内容。
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">学习成果：</th>
```

Comment:
raised [another PR](https://github.com/mdn/translated-content/pull/25706) for this common glossary